src/helpers/helpers.js

```diff
@@ -91,3 +91,36 @@ export const getFilteredProducts = (products, state) => {
   }
   return tempProducts;
 };
+
+export const totalItemsInCart = cart => {
+  return [...cart].reduce((acc, curr) => (acc = acc + curr.qty), 0);
+};
+
+export const getTotalMrpInCart = cart => {
+  return [...cart].reduce(
+    (acc, curr) => (acc = acc + curr.priceInMrp * curr.qty),
```

Comment:
why you're spreading array ???

Reply:
Just to avoid doing any thing on the original array.